examples/tox21/evaluate_models_tox21.sh

```diff
@@ -0,0 +1,17 @@
+set -eu
+
+gpu=-1
+methods=(nfp ggnn schnet weavenet rsgcn)
+results=()
+
+mkdir -p results-evaluation
+for method in ${methods[@]}
+do
+    python train_tox21.py --method ${method} --gpu ${gpu}
+    python predict_tox21_with_classifier.py > log-${method}
+    roc_auc=`cat log-${method} | tail -1 | grep -oP '0\.\d+'`
```

Comment:
I have a concern in this line that it relies on the fact that last line of the `predict_tox21_with_classifier.py` is the evaluation result, which may not be guaranteed after the code `predict_tox21_with_classifier.py` refactored.

Reply:
How about modifying `predict_tox21_with_classifier.py` to save the evaluation result in some text file (json file etc), and load this text file in `plot.py`?